evasquez26/angular4-startup
# Security update for staging deploy

## Story / Task

N/A

## What this PR does?

Configures a username and password prompt when trying to use the app.


### Before 

Anyone could see the app on digital garage

### After

Only users with the correct credentials will be able to see the app

### How to test / reproduce

- Execute `API_URL=<yourapiurl> USERNAME=<yourusername> PASSWORD=<yourpassword> npm run start`
- Try to open the app on `http:localhost:8080`
- It should prompt for username and password

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO
